Rubberduck.Core/Common/ClipboardWriter.cs

```diff
@@ -59,5 +75,48 @@ public void Flush()
                 _data = null;
             }
         }
+
+        public void AppendInfo<T>(ColumnInfo[] columnInfos,
+            IEnumerable<T> results,
+            string title,
+            ClipboardWriterAppendingInformationFormat appendingInformationFormat) where T : IExportable
```

Comment:
You should not need a genetic here, since `IEnumerable<out T>` is covariant. You can simply use `IEnumerable<IExportable>`.